js/ui/select_box.js

```diff
@@ -451,12 +451,26 @@ var SelectBox = DropDownList.inherit({
         return result;
     },
 
+    _isOldDataSourceValue: function() {
+        return this.option("text") && this.option("text") === this.option("value");
```

Comment:
Is it works correctly with the DS contains object items? 
I can define `displayExpr` that represents `text`, and `valueExpr` that represents `value`

Reply:
Thanks. I have check this function and _isOldDataSourceValue is not required